### PR TITLE
Add Prometheus parser and Kubernetes input plugin

### PR DESCRIPTION
**Pull request by titilambert**
Hello !
This plugin does not use k8s client lib. It uses Prometheus client lib.
I also add the Prometheus parser.
@sparrc maybe you prefer I split the PR ? (Prometheus parser/Kubernetes input plugin)

### CODE REVIEW

File: plugins/parsers/prometheus/parser_test.go [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+	assert.Equal(t,
+		map[string]string{"verb": "POST", "resource": "bindings"},
+		metrics[0].Tags())
+
+}
+
+func TestParseLineInvalidPrometheus(t *testing.T) {
+	parser := PrometheusParser{}
+	metric, err := parser.ParseLine(validUniqueLine)
+	assert.NotNil(t, err)
```
Comment (sparrc) [inline plugins/parsers/prometheus/parser_test.go]: change these to just one: `assert.NoError`

File: plugins/parsers/prometheus/parser_test.go [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var exptime = time.Date(2009, time.November, 10, 23, 0, 0, 0, time.UTC)
+
+const (
+	validPrometheus        = "cpu_load_short,cpu=cpu0 value=10 1257894000000000000"
```
Comment (sparrc) [inline plugins/parsers/prometheus/parser_test.go]: this is valid prometheus?

File: plugins/parsers/prometheus/parser_test.go [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+
+func TestParseLineInvalidPrometheus(t *testing.T) {
+	parser := PrometheusParser{}
+	metric, err := parser.ParseLine(validUniqueLine)
+	assert.NotNil(t, err)
+	assert.Nil(t, metric)
+
+}
+
+/*
```
Comment (sparrc) [inline plugins/parsers/prometheus/parser_test.go]: just remove commented tests, or implement?

**Comment by titilambert** [comment]
@sparrc Thanks for your review ! (I think I have to stop 3 AM code ...)

BTW, I feel that plugin I maybe not Kubernetes specific... I could configure prometheus input plugin to get   basically the same result ... maybe I should make more work to get better metric ?

**Comment by sparrc** [comment]
@titilambert I agree I think it would be better to try to improve the prometheus input plugin to get this information, if possible

**Comment by titilambert** [comment]
Closing because of improving prometheus plugin #707

**Pull request closed without merging**